app/services/twilio.rb

```diff
@@ -0,0 +1,19 @@
+class TwilioAPI
+    attr_accessor :client,:account_sid
+    def initialize
+        account_sid=ENV['account_sid']
+        auth_token=ENV['auth_token']
+        @client=Twilio::REST::Client.new(account_sid, auth_token);
+    end
+    def validate_twilio_request?(accountid)
+        return account_sid==accountid
+    end
+    def sendSms(to,body,thread_id)
+        client.messages.create(
+            from:ENV["twilio_number"],
+            to:to,
+            body:body
+          )
+          Message.create_message(to,ENV["twilio_number"],body,thread_id)
```

Comment:
Model level creation should not be performed here. Purpose of this method is to send message only not to create in DB as well.